src/NosCore.GameObject/Character.cs

```diff
@@ -794,14 +797,14 @@ private void GenerateLevelupPackets()
                 groupMember?.SendPacket(groupMember.Group.GeneratePinit());
             }
 
-            SendPacket(Group.GeneratePinit());
+            await SendPacket(Group.GeneratePinit());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)